src/ontology/cl-edit.owl

```diff
@@ -28486,6 +28487,15 @@ SubClassOf(obo:CL_4023181 obo:CL_0000710)
 SubClassOf(obo:CL_4023181 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002139))
 SubClassOf(obo:CL_4023181 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000015658))
 
+# Class: obo:CL_4023187 (koniocellular cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "WikipediaVersioned:Lateral_geniculate_nucleus&oldid=1098375993"^^xsd:string) obo:IAO_0000115 obo:CL_4023187 "A neuron with a small cell body that is located in the koniocellular layer of the lateral geniculate nucleus (LGN).")
```

Comment:
Not sure about all species, but I think there are several layers of K cells in the LGN. Wording of the definition suggests that there is one layer. 
"... located in the koniocellular layer ..."
Perhaps substituting "the" with "a" would be appropriate.

Also consider adding "K cell" as an exact synonym.